src/wazuh_db/wdb.h

```diff
@@ -657,6 +659,16 @@ int wdb_delete_agent_belongs(int id, int *sock);
  */
 int wdb_remove_group_from_belongs_db(const char *name, int *sock);
 
+/**
+ * @brief Resets the connection_status column of every agent.
+ *        If connection_status is pending or connected it will be changed to disconnected.
+ *        If connection_status is disconnected or never_connected it will not be changed.
```

Comment:
Probably we should mention also here that the manager's connection status will be kept as `active` and not modified.

Reply:
Done